src/main/resources/templates/contact/contactUs.ftl

```diff
@@ -0,0 +1,103 @@
+                                        <div style="display: none;">
+                                            <input type="hidden" name="_wpcf7" value="126"/>
+                                            <input type="hidden" name="_wpcf7_version" value="5.1.6"/>
+                                            <input type="hidden" name="_wpcf7_locale" value="en_US"/>
+                                            <input type="hidden" name="_wpcf7_unit_tag"
+                                                   value="wpcf7-f126-p127-o1"/>
+                                            <input type="hidden" name="_wpcf7_container_post"
+                                                   value="127"/>
+                                        </div>
+                                        <p><span class="wpcf7-form-control-wrap full-name"><input
+                                                        type="text" name="full-name" value="" size="40"
+                                                        class="wpcf7-form-control wpcf7-text"
+                                                        aria-invalid="false"
+                                                        placeholder="Your Full Name"/></span></p>
+                                        <p><span class="wpcf7-form-control-wrap email"><input
+                                                        type="email" name="email" value="" size="40"
+                                                        class="wpcf7-form-control wpcf7-text wpcf7-email wpcf7-validates-as-email"
+                                                        aria-invalid="false"
+                                                        placeholder="Your Email Address"/></span></p>
+                                        <p><span class="wpcf7-form-control-wrap your-subject"><input
+                                                        type="text" name="your-subject" value=""
+                                                        size="40" class="wpcf7-form-control wpcf7-text"
+                                                        aria-invalid="false"
+                                                        placeholder="Subject"/></span></p>
+                                        <p><span class="wpcf7-form-control-wrap message"><textarea
+                                                        name="message" cols="40" rows="10"
+                                                        class="wpcf7-form-control wpcf7-textarea"
+                                                        aria-invalid="false"
+                                                        placeholder="Write Here"></textarea></span></p>
+                                        <p><input type="submit" value="Donate Now"
+                                                  class="wpcf7-form-control wpcf7-submit"/></p>
+                                        <div class="wpcf7-response-output wpcf7-display-none"></div>
```

Comment:
Button name doesn't match to the page logic

Reply:
Fixed